packages/manager/src/components/ExtendedAccordion/ExtendedAccordion.test.tsx

```diff
@@ -1,7 +1,7 @@
 import { shallow } from 'enzyme';
 import * as React from 'react';
 
-import CircleProgress from 'src/components/CircleProgress';
+import { CircleProgress } from 'src/components/CircleProgress';
```

Comment:
Unrelated to your work, but I've never been a fan of the spaces between imports

```suggestion
import { CircleProgress } from 'src/components/CircleProgress';
```

Reply:
I'm not against it, we add spaces when destructuring objects too